algrtm/src/components/AlgorithmicMusicSection.js

```diff
@@ -7,14 +7,14 @@ import "../styles/globals.css";
 
 const AlgorithmicMusicSection = () => {
   return (
-    <section className="flex gap-16 px-[6.75rem] py-[5.88rem] text-neutralLight-neutral10 items-center">
+    <section className="flex flex-col xl:flex-row gap-[1.5rem] xl:gap-16 py-[2.75rem] px-[1rem] xl:px-[2.75rem] xl:py-[5.88rem] text-neutralLight-neutral10 items-center">
```

Comment:
Here `xl:flex-row` should be `lg:flex-row` and on small devices, the `px` should be 0, as it currently gives an extra 1rem padding:

![image](https://github.com/Nospoko/algrtm/assets/26410548/7e989bf9-42bf-462e-9bde-770845a007e5)